webrtc.html

```diff
@@ -3284,39 +3284,59 @@ <h2>Attributes</h2>
-                  <code>addTransceiver("audio")</code> on the
-                  <code><a>RTCPeerConnection</a></code> object, except that this
-                  MUST NOT <a>Update the negotiation-needed flag</a>,  and,
-                  provided this does not fail, proceed with createOffer()'s
-                  regular steps.
-                </p>
-                <p>
-                  In all other situations, it will be disregarded.
-                </p>
+                <p>Offer to receive audio even though no audio will be sent.
+                When this dictionary member is present as an option to <a href=
+                "#dom-rtcpeerconnection-createoffer">createOffer</a>, run the
+                <a>offer to receive</a> steps with a <var>mediaType</var> of
+                "audio" instead of the regular <a href=
+                "#dom-rtcpeerconnection-createoffer">createOffer</a> steps.</p>
               </dd>
               <dt><dfn>offerToReceiveVideo</dfn> of type <span class="idlMemberType"><a>boolean</a></span></dt>
               <dd>
-                <p>
-                  Whenever this is given a non-false value, and the
-                  <code><a>RTCPeerConnection</a></code> has no non-stopped
-                  "sendrecv" or "recvonly" video transceivers, createOffer()
-                  MUST as its first step invoke the equivalent of
-                  <code>addTransceiver("video")</code> on the
-                  <code><a>RTCPeerConnection</a></code> object, except that this
-                  MUST NOT <a>Update the negotiation-needed flag</a>,  and,
-                  provided this does not fail, proceed with createOffer()'s
-                  regular steps.
-                </p>
-                <p>
-                  In all other situations, it will be disregarded.
-                </p>
+                <p>Offer to receive video even though no video will be sent.
+                When this dictionary member is present as an option to <a href=
+                "#dom-rtcpeerconnection-createoffer">createOffer</a>, run the
+                <a>offer to receive</a> steps with a <var>mediaType</var> of
+                "video" instead of the regular <a href=
+                "#dom-rtcpeerconnection-createoffer">createOffer</a> steps.</p>
               </dd>
             </dl>
+            <p>To <dfn>offer to receive</dfn> with <var>mediaType</var>, run
+            the following steps:</p>
+            <ol>
+              <li>
+                <p>Let <var>connection</var> be the current <code>
+                <a>RTCPeerConnection</a></code> object.</p>
+              </li>
+              <li>
+                <p>If the value of the dictionary member is false, jump to the
+                step labeled <em>create offer</em>.</p>
+              </li>
+              <li>
+                <p>If <var>connection</var> has any stopped "sendrecv" or
+                "recvonly" transceivers of type <var>mediaType</var>, jump to
```

Comment:
how is the type of a transceiver defined? Both the transceivers sender and receiver have a kind but the transceiver itself does not. Might need an internal slot for the kind?

Reply:
actually the kind is only guaranteed to be on transceiver.receiver.track

Reply:
I think the correct thing to say here is the transceiver kind defined in http://w3c.github.io/webrtc-pc/#dfn-transceiver-kind

Reply:
done in https://github.com/w3c/webrtc-pc/pull/1681